frontend/packages/console-shared/src/components/status/ErrorStatus.tsx

```diff
@@ -1,21 +1,10 @@
 import * as React from 'react';
-import PopoverStatus from './PopoverStatus';
-import StatusIconAndText from './StatusIconAndText';
 import { RedExclamationCircleIcon } from './Icons';
+import { StatusComponentProps } from './types';
+import GenericStatus from './GenericStatus';
 
-type ErrorStatusProps = {
-  title?: string;
-  iconOnly?: boolean;
-  noTooltip?: boolean;
-};
-
-const ErrorStatus: React.FC<ErrorStatusProps> = (props) => {
-  const icon = <RedExclamationCircleIcon />;
-  return props.children ? (
-    <PopoverStatus {...props} icon={icon} />
-  ) : (
-    <StatusIconAndText {...props} icon={icon} />
-  );
-};
+const ErrorStatus: React.FC<StatusComponentProps> = (props) => (
```

Comment:
As the components are reduced to these one-liners, shouldn't it make sense to move them into a single file? @jtomasek , wdyt?

Reply:
Yes, sounds good

Reply:
moved to `index.tsx`